<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgba(252, 252, 252, 255);
}
QLineEdit {
    border: 2px solid #DDD;
    border-radius: 8px;
    padding: 6px;
    background: #FFF;
    color: #111;
    selection-background-color: #DDD;
}

QLineEdit:focus {
    border: 2px solid #111;
    background: #FFF;
}

QPushButton {
color: #111;
background-color: rgb(230, 230, 230);
border-radius: 15px;
border: 2px solid #000;
padding: 8px;
font-weight: bold;
}
QPushButton:hover {
    background-color: rgb(200, 200, 200);
}

QPushButton:pressed {
    background-color: #FFF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="verticalWidget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>18</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="verticalWidget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../icons/online-registration.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLineEdit" name="line_edit_login">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">margin-bottom:15px;</string>
          </property>
          <property name="placeholderText">
           <string>Введите логин</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_edit_password">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Введите пароль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_edit_password_repeat">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhHiddenText</set>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Повторите пароль</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QPushButton" name="but_registration">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Зарегистрироваться</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icons/add_user.png</normaloff>../icons/add_user.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
